stable/redis/values.yaml

```diff
@@ -87,3 +87,7 @@ networkPolicy:
   ##
   allowExternal: true
 
+service:
+  type: ClusterIP
+  annotations: {}
+  loadBalancerIP:
```

Comment:
According to the linter:

>  93:18     error    no new line character at the end of file  (new-line-at-end-of-file)

Please add a new line at the end of the file.

Reply:
Thanks @mattfarina. Done.